### ISSUE TITLE
[BUG] Keyboard Accessibility doesn't work (sort of)

### ISSUE THREAD

**Issue by 72gm** [react-dropzone/react-dropzone#1405]
**Describe the bug**
Being able to open up the react-dropzone using the keyboard only doesn't work when creating your own implementation

**To Reproduce**
Looking at your examples this works fine

**Expected behavior**
I'd expect this to work out the box - do your examples have some bespoke css/js that is only applied to your examples that enable this?

**Comment by rolandjitsu** [comment]
@72gm could you share what you've tried? You can use a fork of this [sandbox](https://codesandbox.io/p/devbox/currying-thunder-jq6d5) to replicate your issue.

There's no custom js/css that's needed. It can be disabled, but it's enabled by default. What I can think of is that something else is preventing the event from propagating or maybe a misuse of this lib.

**Comment by 72gm** [comment]
yeah you're probably right (not sure how though!)... I'll stick up an example soon... in the meantime I think I've managed to workaround it by

importing "open" in the usedropzone set up (and added noKeyboard: false)

```
const { getRootProps, getInputProps, isDragActive, open } = useDropzone({
        accept: accept,
        maxSize: fileMaxSize,
        onDrop,
        multiple: multiFile,
        maxFiles: enforceMaxFiles ? maxFiles : undefined,
        noKeyboard: false
    });
```

then added a keyboard handler to open it

`<div {...getRootProps({ style, className: 'dropzone' })} ref={ref} onKeyDown={(e) => handleDropzoneKeyPress(e)}>
`
 ```
//handle return or space on dropzone
    const handleDropzoneKeyPress = (e: any) => {
        console.log('keyboard')
        if (e.key === 'Enter' || e.key === ' ') {
            open();
        }
    };

```
